src/Result/StorePaymentMethodLightningNetwork.php

```diff
@@ -6,4 +6,9 @@
 
 class StorePaymentMethodLightningNetwork extends AbstractStorePaymentMethodResult
 {
+    public function isEnabled(): bool
```

Comment:
I think this can be moved to `AbstractStorePaymentMethodResult` class if it is used on every pm anyway?

Reply:
@JudithWebdev moving this to the abstract class makes still sense imo, so if you could do that I think we can merge. Or does anything speak against it? cc @woutersamaey

Reply:
I don't mind. If it matches the API specs, then sure, let's do it.

Reply:
As far as I understand it is solely a helper and has nothing to do with the API, just some kind of avoiding code duplication and putting it on the abstract class as every payment method has this field.

Reply:
Well yeah, that's what I meant :+1:

Reply:
@ndeet I moved it to the abstract class, added a getter isNew() for an Invoice as well